src/jit/codegencommon.cpp

```diff
@@ -7298,13 +7298,15 @@ void CodeGen::genProfilingEnterCallback(regNumber initReg, bool* pInitRegZeroed)
         return;
     }
 
-#if defined(_TARGET_AMD64_) && !defined(UNIX_AMD64_ABI) // No profiling for System V systems yet.
+#if defined(_TARGET_AMD64_)
     unsigned   varNum;
     LclVarDsc* varDsc;
 
     // Since the method needs to make a profiler callback, it should have out-going arg space allocated.
     noway_assert(compiler->lvaOutgoingArgSpaceVar != BAD_VAR_NUM);
```

Comment:
Shouldn't this first `noway_assert` also be under the following `#if`? I believe it's only defined under `FEATURE_FIXED_OUT_ARGS` which is only true for amd64.